src/markov/markov_approx.jl

```diff
@@ -130,3 +130,50 @@ function _rouwenhorst(p::Real, q::Real, m::Real, Δ::Real, n::Integer)
         return linspace(m-Δ, m+Δ, n), θN
     end
 end
+
+
+"""
+Accepts the simulation of a discrete state Markov chain and estimates
+the transition probabilities via maximum likelihood.
```

Comment:
Hey @natashawatkins good catch.

This should already be ready to go as we are using Documenter to generate docs and math support seems to be enabled by default.

I think @Shunsuke-Hori  uses this in #163